src/main.cpp

```diff
@@ -453,6 +454,52 @@ class bearer_token_eviction_manager
     } // evict
 }; // class bearer_token_eviction_manager
 
+class oidc_state_eviction_manager
```

Comment:
Can this be combined with the bearer token eviction manager class? Seems like it can.

Reply:
Yes, just separated them for making the purpose of the classes clear.

AKA, didn't want to pollute the original class.

Reply:
I think it's fine to combine them since they are so similar. Branching would essentially be based on the pointer obtained by the any_cast.

What do you think?

Reply:
That also means both types are handled within the same locked mutex. Without that, the two eviction managers may compete on acquiring the process stash mutex.

Thoughts?

Reply:
True on competing items. Can put them together. Means we have one eviction timer, though I don't think that's much of a problem.

Reply:
Removed class, merged with other.